src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/translator.kt

```diff
@@ -107,6 +107,6 @@ private fun IncentiveLevel?.toCurrentIncentive(): CurrentIncentive? = this?.let
   CurrentIncentive(
     level = IncentiveLevel(it.iepCode, it.iepLevel),
     nextReviewDate = it.nextReviewDate,
-    dateTime = it.iepTime
+    dateTime = it.iepTime.withNano(0) // ES only stores to the second
   )
```

Comment:
it gets truncated anyway by ES but by also doing it here is ensures the diff comparisons work